From 4def84da82cea36868e3903436eae8bca2efcdd0 Mon Sep 17 00:00:00 2001
From: Thomas Bourgoin <thomas.bourgoin@foss.st.com>
Date: Thu, 8 Jun 2023 11:37:35 +0200
Subject: [PATCH 0339/1141] arm64: defconfig: enable CRYP support

Enable STM32 CRYP support on 64 bits platform.

Signed-off-by: Thomas Bourgoin <thomas.bourgoin@foss.st.com>
Change-Id: Ic0a76eb97bb0203d997fdecb8801b0b6b089881d
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/311579
Reviewed-by: Yann GAUTIER <yann.gautier@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Domain-Review: Yann GAUTIER <yann.gautier@foss.st.com>
---
 arch/arm64/configs/fragment-02-defconfig-addons.config | 1 +
 1 file changed, 1 insertion(+)

--- a/arch/arm64/configs/fragment-02-defconfig-addons.config
+++ b/arch/arm64/configs/fragment-02-defconfig-addons.config
@@ -7,6 +7,7 @@ CONFIG_BACKLIGHT_GPIO=y
 CONFIG_CAN_M_CAN=m
 CONFIG_CAN_M_CAN_PLATFORM=m
 CONFIG_COUNTER=m
+CONFIG_CRYPTO_DEV_STM32_CRYP=m
 CONFIG_DRM=y
 CONFIG_DRM_BRIDGE=y
 CONFIG_DRM_DW_MIPI_DSI=y
